### ISSUE TITLE
False Negative `mandatory-package` Check

### ISSUE THREAD

**Issue by volodya-lombrozo** [objectionary/lints#234]
I have an XMIR file with the following meta:

```xml
<metas>
  <meta>
    <head>version</head>
    <tail>1.2.3</tail>
  </meta>
</metas>
``` 

I added it to handle [mandatory-version](https://github.com/objectionary/lints/issues/207) check and immediately have got the following error:

```xml
XMIR is incorrect: [[mandatory-package WARNING]:0 The +package meta is mandatory, but is absent]
```

It used to work well because if I had empty package I avoided adding `metas` at all. Now I have non-empty `metas` and empty package. `objectionary/lints` complaints about it. 

Please remove this check or fix it. Otherwise, the checks conflicts with each other.

**Comment by volodya-lombrozo** [comment]
@yegor256 Could you have a look, please?

**Comment by volodya-lombrozo** [comment]
Blocks https://github.com/objectionary/jeo-maven-plugin/pull/962

**Comment by yegor256** [comment]
@volodya-lombrozo I believe, `mandatory-package` complains about the absence of `+package` meta in any case, either `+version` is there or not. This is by design. Or I misunderstand the problem?

**Comment by yegor256** [comment]
@volodya-lombrozo it's legal, but it's better to have it (that's why it's a warning, not an error). Maybe you can use some global package for all objects in JEO, something like `java`. Thus, if there are no packages, all objects are `java.foo`. If it's `org.springframework.Foo`, you make it `java.org.springframework.Foo`.

**Comment by volodya-lombrozo** [comment]
@yegor256 I believe it's better just to remove this check. We do have empty packages. It's perfectly legal. The absence of the package entry doesn't influence anything. It's not an anti-pattern and doesn't break anything. I don't see any reason for leaving the check. 

Moreover, please, pay attention, I don't have `package` entry at all:

```xml
<metas>
  <meta>
    <head>version</head>
    <tail>1.2.3</tail>
  </meta>
</metas>
```

If I would have 
```xml
<metas>
  <meta>
    <head>version</head>
    <tail>1.2.3</tail>
  </meta>
  <meta>
    <head>package</head>
    <tail></tail>
  </meta>
</metas>
```
I would hardly agree (no) that it might be a warning.

**Comment by yegor256** [comment]
@volodya-lombrozo remember, the lints exist not only for XMIR checking, but mostly for EO programmers. We want them to be as disciplined as possible, that's why lints are as strict as we can make them. Having an object without a package means having it in the global scope of visibility (the entire "universe" should see it and use it). This is not a good idea.

**Comment by volodya-lombrozo** [comment]
@yegor256 Maybe I can ignore this check somehow? For XMIR checking I don't need this rule.

**Comment by yegor256** [comment]
@volodya-lombrozo sure, you can add this and the rule will be ignored:

```
<metas>
  <meta>
    <head>unlint</head>
    <tail>mandatory-package</tail>
  </meta>
</metas>
```

**Comment by volodya-lombrozo** [comment]
> @volodya-lombrozo sure, you can add this and the rule will be ignored:
> 
> ```
> <metas>
>   <meta>
>     <head>unlint</head>
>     <tail>mandatory-package</tail>
>   </meta>
> </metas>
> ```

@yegor256 Since we can ignore this check, I believe we can close this issue.
